Source/Rio.Models/DataTransferObjects/ParcelAllocation/ParcelLedgerCreateDto.cs

```diff
@@ -2,19 +2,23 @@
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 
-namespace Rio.Models.DataTransferObjects.ParcelLedgerCreateDto
+namespace Rio.Models.DataTransferObjects.ParcelAllocation

```

Comment:
Why do we have an Allocation reference again?

Reply:
Folder renamed